submissions/benjaminKamena/phpDataase/queryDb.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include('cookie.php');
+
+$results = isset($_COOKIE['customers'])? unserialize($_COOKIE['customers']) : $queryDatabase->getAllCustomers();
+
+try{
```

Comment:
Kindly format this block of code from line 7 - 32

Reply:
what do you mean?